scripts/publish-all-packages.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -x
+set -e
+
+ROOT_DIR="$(dirname "$0")/.."
+
+echo "publishing all packages"
+
+${ROOT_DIR}/scripts/publish-package.sh "core"
```

Comment:
this is where we are going to add the rest of the packages in the future?

Reply:
correct